model/otel/registry.yaml

```diff
@@ -22,6 +22,10 @@ groups:
         brief: "Description of the Status if it has a value, otherwise not set."
         examples: ['resource not found']
         stability: stable
+      - id: otel.is_sampled
```

Comment:
nit: I would prefer including `span` in the name: `otel.span.is_sampled`

Reply:
Agree that including `span` is better, I just wasn't sure whether we're allowed to do it due to consistency with the existing stable `otel.status_code` and `otel.status_description` attributes.

I've renamed it now in 226b42f, we can revert that change if consistency is indeed a problem.